packages/compiler/src/index.js

```diff
@@ -69,13 +71,28 @@ function loadBabelConfig(filename, config) {
     ast: markoConfig.ast
   };
 
-  if (markoConfig.modules === "cjs") {
+  if (markoConfig.stripTypes === undefined) {
+    markoConfig.stripTypes =
+      markoConfig.output !== "source" && markoConfig.output !== "migrate";
+  }
+
+  if (markoConfig.stripTypes) {
     requiredPlugins.push([
-      require.resolve("@babel/plugin-transform-modules-commonjs"),
-      { loose: true }
```

Comment:
Does losing this have any effect?

Reply:
Shouldn't have an impact to my knowledge. Inlining the import does simplify bundling the Marko compiler though.

Reply:
To be clear this line just moved and had I now directly import the plugin (the diff is bad) https://github.com/marko-js/marko/pull/1826/files#diff-24715af10c4aae7713530fe70bcafd96e43d8323c1e62c7bfb7af916a0766f3aR92-R95